srcalc/src/main/java/gov/va/med/srcalc/domain/model/MultiSelectOption.java

```diff
@@ -1,52 +1,78 @@
 package gov.va.med.srcalc.domain.model;
 
+import gov.va.med.srcalc.util.Preconditions;
+
 import java.util.Objects;
+import java.util.regex.Pattern;
+
 import javax.persistence.*;
 
 /**
- * <p>An option for a {@link MultiSelectVariable}.</p>
+ * <p>An option for a {@link MultiSelectVariable}. Simply wraps a String value,
+ * but enforces constraints on the String.</p>
  * 
  * <p>Per Effective Java Item 17, this class is marked final because it was not
  * designed for inheritance.</p>
  */
-@Entity
+@Embeddable
 public final class MultiSelectOption
 {
-    private int fId;
-    private String fValue;
+    /**
+     * The maximum length of a valid value: {@value}.
+     * @see #getValue()
+     */
+    public static final int VALUE_MAX = 80;
     
-    public MultiSelectOption()
-    {
-    }
+    /**
+     * A regular expression that defines a valid value. Same as {@link
+     * Variable#VALID_DISPLAY_NAME_REGEX}.
+     */
+    public static final String VALID_VALUE_REGEX = Variable.VALID_DISPLAY_NAME_REGEX;
     
-    public MultiSelectOption(final String value)
-    {
-        fValue = value;
-    }
+    /**
+     * English description of the valid value characters for readable error
+     * messages.
+     * @see #VALID_VALUE_REGEX
+     */
+    public static final String VALID_VALUE_CHARACTERS = Variable.VALID_DISPLAY_NAME_CHARACTERS;
+    
+    /**
+     * Precompiled version of {@link #VALID_VALUE_REGEX} for efficiency.
+     */
+    public static final Pattern VALID_VALUE_PATTERN = Pattern.compile(VALID_VALUE_REGEX);
+    
+    private String fValue;
     
     /**
-     * The object's surrogate primary key. Don't show this to the user.
+     * Constructs an instance with dummy values for the basic properties.
+     * Intended for bean construction only.
      */
-    @Id
-    public int getId()
+    MultiSelectOption()
     {
-        return fId;
+        fValue = "unset";
     }
-
+    
     /**
-     * For reflection-based construction only. Business code should never modify
-     * the surrogate key as it is generated from the database.
+     * Creates an instance with the value filled.
```

Comment:
The English is weird here. Maybe "Creates an instance from the given value." would be better.